2024/survey.yaml

```
@@ -253,11 +259,16 @@ questions:
       - VBA
       - Visual Basic (.Net)
       - Zig
-  - prompt: Do you use NixOS?
-    type: single
+  # Both query information about NixOS usage and the way people install Nix
```

```suggestion
```

as we delete the "Do you use NixOS?"

---

I assumed deleting the “Do you use NixOS” wouldn't loose any information since NixOS users would check the `NixOS` box in “How did you install Nix?”

Does the below make it clearer?

```suggestion
  # Query information about the installation method.
  # Also gives insight on the proportion of NixOS users since these would be the one who checked the `NixOS` box
```